aws-lambda-java-core/src/main/java/com/amazonaws/services/lambda/runtime/CustomPojoSerializer.java

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+
+package com.amazonaws.services.lambda.runtime;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import java.lang.reflect.Type;
+
+/**
+ * Interface required to implement a custom plain old java objects serializer
+ */
+public interface CustomPojoSerializer {
+
+    /**
+     * Deserializes from input stream to plain old java object
+     * @param input input stream
+     * @param type plain old java object type
+     * @return deserialized plain old java object of type T
```

Comment:
What's `T`?

Reply:
It's the type of the deserialized instance

Reply:
I see the intention. But "deserialized plain old java object of type T" uses `T` which is undefined and unused anywhere else.